packages/echo-core/src/hooks/useInternalLink.ts

```diff
@@ -35,7 +34,8 @@ export function useInternalLink(): Link {
         }
 
         eventTracker && eventTracker('InternalLink', 'Opened', { linkTo: linkTo });
-        history.push(linkTo + params);
+        const linkParams = params ? params : '';
+        history.push(linkTo + linkParams);
```

Comment:
For å unngå forvirringer, tror vi bør ta vekk hele linkParams, og la det være del av linkTo heller. Vi kan enkelt ta en string.split her for kun å logge første del til appInsight, dette er jo eneste grunn til det kommer inn som 2 parameteres idag.

Og online bør renames til isInternetRequired som er lettere å forstå. Online er litt mer diffust hva betyr.